readme.md

```diff
@@ -92,7 +93,8 @@ Using Ruby, instantiate an array called `fruits` that contains `apple`, `banana`
 Then, use an enumerator to print to the console the sentence "I'd like to eat a [fruit]" once for each fruit.
 
 ```rb
-# Your answer...
+var fruits = ["apple", "banana", "orange"];
+puts : "I'd like to eat a + [fruits] ."
 
 ```
 
```

Comment:
You instantiated the array but there's no enumerator. The way you have it written out now, you're trying to print out the entire array at once. Although there are some syntax errors here, so it would error out.

Easiest way to do what the question is asking would be with an `.each` statement. So it would look something like this...

fruits = [ "apple", "banana", "orange" ]
fruits.each do |fruit|
  puts "I'd like to eat a #{fruit}"
end
```